ts/store/reducers/entities/services/index.ts

```diff
@@ -63,6 +64,37 @@ const reducer = combineReducers<ServicesState, Action>({
 // Selectors
 export const servicesSelector = (state: GlobalState) => state.entities.services;
 
+// A selector to get the organizations selected by the user as areas of interests
+export const organizationsOfInterestSelector = createSelector(
+  [userMetadataSelector, servicesSelector],
+  (potUserMetadata, services) => {
+    const visibleServices = services.visible;
+    // If the user never select areas of interest, return an undefined object
+    return pot.toUndefined(
+      pot.map(
+        potUserMetadata,
+        _ =>
+          // filter organization by selecting those ones have
+          // at least 1 visible service inside
+          _.metadata.organizationsOfInterest && pot.isSome(visibleServices)
+            ? _.metadata.organizationsOfInterest.filter(org => {
+                const organizationServices = services.byOrgFiscalCode[org];
+                return (
+                  organizationServices &&
+                  visibleServices.value.findIndex(
+                    s =>
+                      organizationServices.findIndex(
+                        os => os === s.service_id
+                      ) !== -1
+                  )
```

Comment:
```suggestion
                  ) !== -1
```
I lost my mind to find this little bug (I bug that I suggested :weary:)
If you want to reproduce try to add both "Città di Torino". In locals will be displayed only one of them

Reply:
updated https://github.com/teamdigitale/io-app/pull/1327/commits/cc6459856917c06cf138c589846994476912fceb!